Makefile

```diff
@@ -48,10 +46,10 @@ pyflakes3:
 	@$(CWD)/tools/run-pyflakes3
 
 unittest: clean_pyc
-	nosetests $(noseopts) tests/unittests cloudinit
+	python -m pytest -v tests/unittests cloudinit
```

Comment:
Given that we are dropping Python 2 support with 20.1 release, I think the `unittest` target should be dropped rather than changing it.